tests/PhpSpreadsheetTests/WorksheetTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests;
+
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet;
+use PHPUnit_Framework_TestCase;
+
+class WorksheetTest extends PHPUnit_Framework_TestCase
+{
+    public function testSetTitle()
+    {
+        $test_title = str_repeat('a', 31);
+
+        $worksheet = new Worksheet();
+        $worksheet->setTitle($test_title);
+        $this->assertSame($test_title, $worksheet->getTitle());
+    }
+
+    public function setTitleInvalidProvider()
+    {
+        return [
+            [str_repeat('a', 32), 'Maximum 31 characters allowed in sheet title.'],
+            ['invalid*title', 'Invalid character found in sheet title'],
+        ];
+    }
+
+    /** @dataProvider setTitleInvalidProvider */
```

Comment:
`/** ... */` is [standard](https://www.phpdoc.org/docs/latest/guides/docblocks.html#anatomy-of-a-docblock), but I think what the code style is choking on is the presence of a docblock that does not document  all parameters.

Regardless, should be fixed now.